index.html

```diff
@@ -5,9 +5,13 @@
   <title>&lt;juicy-popover&gt;</title>
 
   <link rel="stylesheet" href="http://juicy.github.io/github-markdown-css/github-markdown.css">
-
+  
   <!-- Importing Web Component's Polyfill -->
+  <!-- v0 with Shadow DOM-->
   <script src="../webcomponentsjs/webcomponents.js"></script>
+  <!-- or v1 -->
+  <script src="../webcomponentsjs/webcomponents-lite.js"></script>
+
```

Comment:
I think these lines are not needed, because `bower install` does not get v0 anyway:

```
<!-- v0 with Shadow DOM-->
<script src="../webcomponentsjs/webcomponents.js"></script>
```